### ISSUE TITLE
dowload all 

### ISSUE THREAD

**Issue by cometta** [Azure/azure-storage-fuse#285]
Is there any parameter I can use when calling blobfuse command so that it will immediately download all the files from azure into local machine before I started to access the files (like reading content of one of the files)?

**Comment by amnguye** [comment]
Once you mount the container, it should be synced to your container in azure. So you should be able to see all your files/folders and open them and they should be concurrent with what you have in azure.

Did you mean after you've already called blobfuse and mounted the container?

**Comment by cometta** [comment]
@amnguye, ya, i able to see the files/folders after mounted, called blobfuse. I think files are only started to sync/download once there is open+read to each of the file right?

**Comment by amnguye** [comment]
Yes. Until you open the file, then blobfuse will download the file for you to view and open and read.

Is there a reason why you would want to download all the files right after you've mounted your container? If what you're going for is just downloading everything from your container, you should look into AzCopy

**Comment by amnguye** [comment]
Has your question been answered or is this still an issue? If it has been resolved please close the issue.
